docs/source/architecture.rst

```diff
@@ -0,0 +1,15 @@
+Architecture
+============
+
+
+Data ingest
+-----------
+
+Data are pulled in with a number of celery tasks, listed at :ref:`wikidata-tasks`.
+
+
+Boundaries
+----------
+
+``commons-api`` embeds `represent-boundaries <https://represent.poplus.org/>`_ for storing boundary data. Boundaries are
```

Comment:
I'm guessing this sentence will be finished in a future PR? 🙂

Reply:
Oops, yes :-)